os_stub/mbedtlslib/include/mbedtls/config.h

```diff
@@ -859,7 +859,7 @@ extern void my_free( void *ptr );
  *        MBEDTLS_ECP_ALT, MBEDTLS_ECDH_XXX_ALT, MBEDTLS_ECDSA_XXX_ALT
  *        and MBEDTLS_ECDH_LEGACY_CONTEXT.
  */
-/*#define MBEDTLS_ECP_RESTARTABLE*/
+#define MBEDTLS_ECP_RESTARTABLE
```

Comment:
Would you please explain why this is needed now?

Reply:
> Would you please explain why this is needed now?

If MBEDTLS_ECP_RESTARTABLE is not defined then access to MBEDTLS_ECDH_LEGACY_CONTEXT is disabled in mbedtls/include/mbedtls/ecdh.h, This forces to us to enable MBEDTLS_ECP_RESTARTABLE. Also noticed MBEDTLS_ECDH_LEGACY_CONTEXT is removed from mbedtls_config.h